bin/nib

```diff
@@ -126,25 +155,22 @@ if __name__ == '__main__':
             print('Done')
 
         elif options.action == 'wizard':
-            empty = True
-            for dirpath, dirs, files in os.walk('.'):
-                if len(dirs) > 0 or len(files) > 0:
-                    empty = False
-                    break
-
-            overwrite = input('This directory is not empty, the wizard will overwrite existing files. Proceed? [y/N]: ')
-            if not overwrite.lower().startswith('y'):
-                sys.exit(1)
+            # If the directory's not empty
+            if os.listdir('.') != []:
```

Comment:
It would be more succinct; not sure it's clearer.  I prefer the explicit comparison because it lets anyone who isn't sure what os.listdir() returns for an empty directory clearly see that it returns an empty list which not a boolean which is implied by the shorter code.   It all evaluates the same, your call.